.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.env
+tsconfig.json
```

Comment:
The `tsconfig.json` should not be in .gitignore - it is a shared configuration file that is needed by all developers interacting with the repository

Reply:
fixed